packages/browser/src/core/user/index.ts

```diff
@@ -183,18 +203,123 @@ export interface CookieOptions {
   sameSite?: string
 }
 
+export class UniversalStorage<Data extends StorageObject = StorageObject> {
+  private stores: Store[]
+
+  constructor(stores?: Store[]) {
+    this.stores = stores || []
+  }
+
+  private getStores(storeTypes: StoreType[] | undefined): Store[] {
+    return storeTypes
+      ? this.stores.filter((s) => storeTypes.indexOf(s.getType()) !== -1)
+      : this.stores
+  }
+
+  public push(store: Store) {
```

Comment:
Is `push` public so that anyone can write their own implementation and add it?

Reply:
Good catch, removed it as it is not used anywhere. I think if we see a need to add an storage on the fly after creation of a UniversalStorage object we can add it back.